pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
+
+`stack build yesod-bin cabal-install --install-ghc`
+
+- Start development server:
+
+`stack -- exec yesod devel`
```

Comment:
Yes, the `yesod` tool needs to be invoked through the `stack` tool so it can used it context. It's the canonical way to use it, and unfortunately is exactly what makes it cumbersome to get inline help and remember how to use it.

The `stack` tool is the package manager for Haskell, and deserves another (separate) *TLDR* page, I will write one as well and propose it in a separate PR.